src/matching.py

```diff
@@ -254,17 +254,17 @@ def match_display_title(plans: list, post):
     @staticmethod
     def request_plan_list(plans: list, post):
         """
-        Matches strictly to a request for the full list of all known plans
+        Matches strictly to a request at the end of a line for the full list of all known plans
```

Comment:
Ah, if you mean end of line, that's `$`. https://stackoverflow.com/questions/22519318/regex-differences-between-and-a-z

We should probably allow for whitespace and punctuation as well, before the end of line, in case they add a period or something.

`r"show me the plans\W*$"`